04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,69 @@
+const { query } = require('express')
```

Comment:
Requires change:
Dead import: `const { query } = require('express');` that is not being used. This import can be removed.